apps/mobile/mobile-app/components/StyledText.tsx

```
@@ -2,4 +1,249 @@
+              <Text style={styles.statLabel}>Investors</Text>
+            </View>
+            <View style={styles.stat}>
+              <Text style={styles.statValue}>{item.stats.minInvestment}</Text>
+              <Text style={styles.statLabel}>Min. Investment</Text>
+            </View>
+          </View>
+
+          {/* Tags Section */}
+          <View style={styles.tagsContainer}>
+            {item.tags.map((tag, index) => (
+              <Text key={index} style={styles.tag}>{tag}</Text>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)